hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1612,12 +1612,18 @@ enum ResponseCodeEnum {
     /**
     * The provided fee schedule key contains an invalid key.
     */
-    INVALID_FEE_SCHEDULE_KEY = 374;
+    INVALID_FEE_SCHEDULE_KEY = 373;
```

Comment:
What is the difference between this code and `INVALID_CUSTOM_FEE_SCHEDULE_KEY = 247;`? It seems like this one is duplicative.

Reply:
Yep, it seems to be duplicative. Removed 👍